File: php/scripts/cost_centers.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$costCenters = $travelperk->costCenters()->costCenters()->all();
write_output("cost_centers", [
	$costCenters->offset,
	$costCenters->limit,
	count($costCenters->costCenters),
	$costCenters->costCenters[0]->id,
	$costCenters->costCenters[0]->name,
	$costCenters->costCenters[0]->countUsers,
]);

File: python/scripts/cost_centers.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
cost_centers = travelperk.cost_centers().cost_centers().all()
write_output("cost_centers", [
    cost_centers.offset,
    cost_centers.limit,
    cost_centers.cost_centers,
    cost_centers.cost_centers[0].id,
    cost_centers.cost_centers[0].name,
    cost_centers.cost_centers[0].count_users,
])
